product_docs/docs/edbcloud/beta/using_cluster/connecting_your_cluster/01_connecting_from_azure/02_virtual_network_peering.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Virtual Network Peering Example
+---
+
+These are the steps to connect using Virtual Network Peering. To walk you through this example in your own environment, see the prerequisites described in [Connection Walkthrough Examples](../../connecting_your_cluster/#connection-walkthrough-examples).
```

Comment:
I'm personally not crazy about linking people over to the Connection Walkthrough Examples section in every one of these documents. It's only a few lines of text and forces the customer to jump around just to achieve what they're trying to achieve.

Reply:
Moved to content to the beginning of each example.